src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Please avoid using 'eslint-disable' unless necessary. In this case, you can replace 'any' with a more specific type or use 'unknown' if the type is truly unknown 🧐